docsbox/docs/tasks.py

```diff
@@ -12,26 +12,42 @@
 from docsbox.docs.via_controller import *
 
 
-def get_task(task_id):
+def get_task(task_id: str):
     queue = rq.get_queue()
     return queue.fetch_job(task_id)
 
+def remove_file(path: str):
+    os.remove(path)
 
-def remove_file(path):
-    return os.remove(path)
+class FileInfo:
+    file_path: str
+    filename: str
+    file_id: str
+    mimetype: str
+    pdf_version: str
+    datetime: str
 
+class FileInfoException:
+    has_failed: bool
```

Comment:
could we create a __init__ method and populate the class using it instead of field by field?

Reply:
Done, except document related ones since it doesn't make sense to them